### PR TITLE
WIP Fixes a case where JSON.parse would try to parse undefined.

### PR DESCRIPTION
**Pull request by EstherDarkish**

### CODE REVIEW

File: src/components/graph/menus/modification-list-item.js [hunk 1]
```diff
@@ -77,7 +77,9 @@ export const ModificationListItem = ({
         finally, we uses the default value of equipmentId or empty string
     */
     const getComputedLabel = useCallback(() => {
-        const modificationMetadata = JSON.parse(modif.messageValues);
+        const modificationMetadata = modif?.messageValues
+            ? JSON.parse(modif.messageValues)
+            : {};
```
Comment (Mathieu-Deharbe) [inline src/components/graph/menus/modification-list-item.js:82]: If {} is affected to modificationMetadata won't you have some similar trouble in the following switch ? (return modificationMetadata.lineToSplitId etc)

**Comment by EstherDarkish** [comment]
This is maybe not the way to solde this issue. Changed the PR to WIP.

**Review by SlimaneAmar — CHANGES_REQUESTED** [review]
Message value is never null !

**Comment by EstherDarkish** [comment]
Not the right way to fix the underlying issue.

**Pull request closed without merging**